docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18903,6 +18903,555 @@
             }
           },
           "name": "Deploy-Storage-sslEnforcement"
+        },
+        {
+          "properties": {
+            "displayName": "Audit-MachineLearning-PrivateEndpointId",
+            "mode": "Indexed",
+            "description": "Audit public endpoints that are created in other subscriptions for machine learning.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Machine Learning"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
```

Comment:
Having a deny here, is problematic for some services, because this would mean that no managed private endpoints could be created in Synapse and Data Factory. I would advise to move forward with an Audit effect, but I can definitely also add a Deny effect and then the customer has to leverage exemptions, for the Data Platform Landing Zones. What do you think @krnese ?

Reply:
Include it as part of the allowed values for the parameter, and that should be fine

Reply:
DONE